cmd/metrics-server/app/options/options.go

```diff
@@ -44,9 +44,10 @@ type Options struct {
 	KubeletClient  *KubeletClientOptions
 	Logging        *logs.Options
 
-	MetricResolution time.Duration
-	ShowVersion      bool
-	Kubeconfig       string
+	MetricResolution      time.Duration
+	RequestKubeletTimeout time.Duration
```

Comment:
This field should be part of `KubeleClientOptions`

Reply:
ok, I will update it